src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The Ui class deals with the interactions with the user.
+ */
+public class Ui {
+    private Scanner sc;
```

Comment:
Perhaps you can use a more descriptive name such as scanner?

Reply:
Alright thanks